packages/ckeditor5-engine/src/model/history.js

```diff
@@ -18,10 +20,10 @@ export default class History {
 		/**
 		 * Operations added to the history.
 		 *
-		 * @protected
-		 * @member {Array.<module:engine/model/operation/operation~Operation>} module:engine/model/history~History#_operations
+		 * @readonly
+		 * @type {Array.<module:engine/model/operation/operation~Operation>}
 		 */
-		this._operations = [];
+		this.operations = [];
```

Comment:
I know that I suggested this change in the original issue, but now as I think of it, I am not sure if it is needed. In places where we use `_operations` maybe we can use `getOperations()`. It will be faster than now (won't need to look through all ops).

Reply:
We iterate through `operations` from the end in RTC Client. Using the `getOperations()` would slow down it significantly, but maybe it's not a problem.